modules/sdk-core/src/bitgo/utils/mpcUtils.ts

```diff
@@ -169,6 +169,10 @@ export abstract class MpcUtils {
       }
     }
 
+    if (baseCoin.getFamily() === 'ada' && params.unspents) {
+      baseIntent.unspents = params.unspents;
+    }
```

Comment:
Agree with @margueriteblair , why is the user not supplying intents in the intent directly?